main.tf

```diff
@@ -51,6 +53,31 @@ data "aws_iam_policy_document" "default" {
   }
 }
 
+data "aws_iam_policy_document" "ssm_patch_s3_log_policy" {
+  count = local.ssm_path_log_bucket_enabled ? 1 : 0
+  statement {
+    sid = ""
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:PutObjectAcl",
+      "s3:GetEncryptionConfiguration",
+    ]
+    resources = [
+      "arn:aws:s3:::${var.ssm_patch_manager_s3_log_bucket}/*",
+      "arn:aws:s3:::${var.ssm_patch_manager_s3_log_bucket}",
+    ]
+  }
+}
+
+resource "aws_iam_policy" "ssm_patch_s3_log_policy" {
+  count       = local.ssm_path_log_bucket_enabled ? 1 : 0
+  name        = "ssm_patch_manager_log_bucket_policy"
```

Comment:
use `label` module.
we should not use arbitrary names for any resources since it's not unique, not scalable, and could potentially collide with other names (e.g. in diff module/component).
If you need to add more `attributes`, use another `label` module, inherit from `module.this` and add more attributes